module2.test.js

```diff
@@ -32,8 +32,13 @@ describe('Module 2 Answers', function() {
         return 'The price is ' + this.price
       }
     };
+    spyOn(car, 'sayPrice').and.callThrough();
```

Comment:
I'd only add a comment here explaining what this is or telling students not to worry about it (unless they should know?)

Reply:
We haven't really talked much about test doubles, and Ali put explanations of mocking further down; that's a good idea, thanks.